frontend/packages/kubevirt-plugin/src/k8s/requests/cdi-upload/cdi-upload-requests.ts

```diff
@@ -67,6 +67,11 @@ const createUploadToken = async (pvcName: string, namespace: string): Promise<st
 export const createUploadPVC = async (dataVolume: V1alpha1DataVolume) => {
   const dataVolumeName = getName(dataVolume);
   const namespace = getNamespace(dataVolume);
+  // BZ-1915694: uploaded DV must have "storage.bind.immediate.requested" annotation.
+  dataVolume.metadata.annotations = {
```

Comment:
we will need the same for `add-template-source.tsx`

Reply:
`add-template-source.tsx` calls this function in case of upload, so it's safe